submissions/oyetejuOloyede/css/style.css

```diff
@@ -0,0 +1,199 @@
+*{
+    box-sizing:border-box;
+}
+
+body{
+    font: normal 16px Arial, Helvetica, sans-serif;
```

Comment:
You should adjust this to `font-size` and `font-family`

Reply:
font and font-family adjusted